tests/Google/Ads/GoogleAds/Lib/V3/GoogleAdsClientBuilderTest.php

```diff
@@ -309,6 +301,14 @@ public function testBuildWithValidTransports(string $transport)
         $this->assertSame($transport, $googleAdsClient->getTransport());
     }
 
+    static public function provideValidTransports()
```

Comment:
Nit: Let's use `public static` for consistency.

Reply:
Good catch, it is required to be PSR-12 compliant!